python/cuspatial/cuspatial/core/binpreds/feature_covers.py

```diff
@@ -36,21 +47,65 @@ class CoversPredicateBase(EqualsPredicateBase):
     pass
 
 
+class LineStringLineStringCovers(BinPred):
```

Comment:
I can't comment on non-changed lines, but please don't add documentation that will go stale quickly, e.g. "in this initial release". Also, the fact that it is implemented using only the equals predicate is an implementation detail, not a feature of the API.  Public documentation should cover features and usage of the API, not implementation details.

Reply:
Thanks for the clarification, those docs and others definitely needed a cleanup.